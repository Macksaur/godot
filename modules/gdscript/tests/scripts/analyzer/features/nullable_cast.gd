func test():
    var nullable_string: String? = "123"
    print((nullable_string as String).begins_with("1"))
    nullable_string = null
    print((nullable_string as String?)?.begins_with("1"))
