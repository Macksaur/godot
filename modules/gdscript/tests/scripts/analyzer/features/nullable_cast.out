GDTEST_OK
true
<null>
